[DOCS] Update attributes and text references for Elastic Defend integration (#2476)
elastic/security-docs: docs/troubleshooting/management/ts-management.asciidoc

.Unhealthy {agent} status
[%collapsible]
====
In some cases, an `Unhealthy` {agent} status may be caused by a failure in the {elastic-defend} integration policy. In this situation, the integration and any failing features are flagged on the agent details page in {fleet}. Expand each section and subsection to display individual responses from the agent.

TIP: Integration policy response information is also available from the *Endpoints* page in the {security-app} (*Manage* -> *Endpoints*, then click the link in the *Policy status* column).

> [role="screenshot"]

Reviewer: This image still shows the old cloud integration in the agent policy. Feel free to ignore this comment if you're already tracking this :)

Author: Screenshot updated!

> To restart a transform that’s not running:

Reviewer: The screenshot above these steps (endpoints-transform-failed.png) looks outdated. It's still showing `endpoint security` in the Endpoints page description.

Author: Screenshot updated!

image::images/transforms-start.png[Transforms page with Start option selected]

. On the confirmation message that displays, click *Start* to restart the transform.

> . The transform’s status changes to `started`. Refresh the page if you don't observe the change.

Reviewer 2: Alt suggestion to `observe`:
```suggestion
. The transform’s status changes to `started`. If it doesn't change, refresh the page.
```
